### DISCUSSION TITLE
ReactNode를 컴포넌트 외부에서 선언해서 사용할 때 꿀팁 (render 메소드로 감싸기) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#37] [Announcements]
# ReactNode를 컴포넌트 외부에서 선언해서 사용할 때 꿀팁 (render 메소드로 감싸기) - 정현수 기술 블로그

ReactNode를 외부에서 사용할 땐 render 메소드로 사용하면 좋아요.

https://junghyeonsu.com/posts/when-use-react-node-outside/

**Comment by yusunghyun** [comment]
render는 클래스형에서 쓴다라는 정해진 사고의 틀에 갖혀있었다는 좋은 생각거리가 되는 글 이네요! 좋은 글 감사합니다! 한번 써보고 팀내 PR을 올려보려구요!
